code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -76,6 +76,12 @@ butcher
 	can_be_adjacent_to_target = FALSE
 	max_dist = 0
 
+	New()
+		..()
+		var/datum/aiTask/succeedable/move/tmp = subtasks[subtask_index]
```

Comment:
plz don't call variables `tmp` :(

Reply:
but tmp is their function. they are tmp. (will fix)